utils/das_data_sorter.py

```diff
@@ -1,6 +1,79 @@
 import pandas as pd 
 import argparse 
 from math import ceil
+from numpy import median, mean
+
+# averages data within the seconds or minutes, according to new_time array
+def avg_Data(array):
+    avg_array = []
+
+    for millisecond_indices in new_time:
+        avg = mean(array[millisecond_indices])
+        avg = round(avg, ndigits=2)
+        avg_array.append(avg)
+
+    return avg_array
+
+# 3 point moving mean smoothing
```

Comment:
When you have a comment which is documenting a function, we usually use python's multiline comments, i.e.
```python
def mean_smooth(data):
""" 3 point moving mean smoothing """
    new_data = [] # etc.
```
If you do your function documentation like this, whatever editor will recognise it and display it as the description of a function when you mouse over it (for example)
![image](https://user-images.githubusercontent.com/17876556/79969695-2e6cbc80-84d5-11ea-8417-a8a0fa7128c2.png)